vuepress/docs/next/tutorials/consume/external-db-ms.md

```diff
@@ -0,0 +1,203 @@
+
+### Adapt the Bundle for Multitenancy
+
+1. In your bundle descriptor `entando.json` microservice section, set `dbms`: `none` to prevent the operator from configuring the internal DB.
+2. For each microservice that is using an external DB, add an `env` section with the Spring username, password, URL, and driver as required for the connection.
+
+The following `entando.json` shows the specs required for a sample microservice named YOUR-MS-SQL: 
+
+```json
+{
+  "microservices": [
+        {
+            "name": "YOUR-MS-SQL",
+            "dbms": "none",
+            "env": [
+                {
+                    "name":"SPRING_DATASOURCE_USERNAME",
+                    "valueFrom": {
+                        "secretKeyRef": {
+                            "name": "ed1d60fc-285a480f-external-books-service-mysql-secret",
+                            "key": "SPRING_DATASOURCE_USERNAME"
+                        }
+                    }
+                },
+                {
+                    "name":"SPRING_DATASOURCE_PASSWORD",
+                    "valueFrom": {
+                        "secretKeyRef": {
+                            "name": "ed1d60fc-285a480f-external-books-service-mysql-secret",
+                            "key": "SPRING_DATASOURCE_PASSWORD"
+                        }
+                    }
+                },
+                {
+                    "name":"SPRING_DATASOURCE_URL",
+                    "valueFrom": {
+                        "secretKeyRef": {
+                            "name": "ed1d60fc-285a480f-external-books-service-mysql-secret",
+                            "key": "SPRING_DATASOURCE_URL"
+                        }
+                    }
+                },
+                {
+                    "name":"SPRING_JPA_DATABASE_PLATFORM",
+                    "value": "org.hibernate.dialect.MySQL8Dialect"
+                }
+            ]
+        }
+    ]
+    ...
+    "name": "YOUR-BUNDLE-NAME",
+    "version": "1.0.0",
+    "description": "a description",
+    "type": "bundle"
+}
+```
+3. Pack and publish your bundle:
+```
+  ent bundle pack
+  ent bundle publish
+```
+
+### Create and Apply a Plugin Configuration Secret
+Prior to installing a bundle, a plugin configuration Secret must be created to provide a different datasource for each tenant. Create a Secret for each microservice in the bundle to provide a series of values required by the Spring Dynamic Datasource Routing for every tenant as needed.  
+
+1. Retrieve the bundle ID and the plugin code:
+ ```
+ ent ecr get-bundle-id docker://registry.hub.docker.com/YOUR-ORG/YOUR-BUNDLE
+ ```
+ ```
+ ent ecr get-plugin-code YOUR-ORG/YOUR-MS-NAME --repo=docker://registry.hub.docker.com/YOUR-ORG/YOUR-BUNDLE
```

Comment:
the bundle ID is not required in this situation since the plugin-code does already contains it. For example:
- `ent ecr get-bundle-id docker://registry.hub.docker.com/entandodemo/agenda-bundle-template` will result into `1619b964`
- `ent ecr get-plugin-code YOUR-ORG/YOUR-MS-NAME --repo=docker://registry.hub.docker.com/entandodemo/agenda-bundle-template` will result into `pn-1619b964-e21d8c94-entandodemo-agenda-bundle-template`

Can we streamline to only the plugin code instruction?